src/Shared/Shared.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
It should probably be "netstandard", since it's a library. I don't _think_ it needs to be reference app, but it might fail, if so, leave it.